NetworkPkg/NvmeOfDxe/NvmeOfConfigStrings.uni

```diff
@@ -0,0 +1,71 @@
+#string STR_RETURN_MAIN_FORM                      #language en-US "Back to Previous Page"
+#string STR_SAVE_CHANGES                          #language en-US "Save Changes"
+#string STR_SAVE_CHANGES_HELP                     #language en-US "Must reboot system manually for changes to take place."
+#string STR_NVMEOF_ATTEMPT_FORM_TITLE             #language en-US "Attempt Configuration"
+#string STR_NVMEOF_ATTEMPT_NAME                   #language en-US "NVMe-oF Attempt Name"
+#string STR_NVMEOF_ATTEMPT_NAME_HELP              #language en-US "The human name defined for this attempt."
+#string STR_SELECT_MAC                            #language en-US "Network Device List"
+#string STR_SELECT_MAC_HELP                       #language en-US "Select the network device to use for this NVMe-oF Attempt configuration from available hardware MAC addresses."
+#string STR_NVMEOF_MAC_STRING                     #language en-US "NVMe-oF Network Device:"
+#string STR_NVMEOF_CONFIG_RETRY                   #language en-US "Connection Retry Count"
+#string STR_NVMEOF_CONFIG_RETRY_HELP              #language en-US "The minimum value is 0 and the maximum is 16. 0 means no retry."
+#string STR_IP_MODE_PROMPT                        #language en-US "Internet Protocol"
+#string STR_IP_MODE_HELP                          #language en-US "IP4, IP6, Autoconfigure. In Autoconfigure mode, NVMe-oF driver will attempt to connect NVMe-oF subsystem via IPv4 stack, if failed then attempt IPv6 stack."
+#string STR_IP_MODE_IP4                           #language en-US "IP4"
+#string STR_IP_MODE_IP6                           #language en-US "IP6"
+#string STR_IP_MODE_AUTOCONFIG                    #language en-US "Autoconfigure"
+#string STR_NVMEOF_ENABLE_DHCP                    #language en-US "Enable DHCP"
+#string STR_NVMEOF_LOCAL_IP_ADDRESS               #language en-US "  Host IP Address"
+#string STR_NVMEOF_LOCAL_MASK                     #language en-US "  Host Subnet Mask"
+#string STR_NVMEOF_LOCAL_GATEWAY                  #language en-US "  Gateway"
+#string STR_NVMEOF_IP_ADDRESS_HELP                #language en-US "Enter IP address in dotted-decimal notation."
+#string STR_NVMEOF_ENABLE_DHCP_ON_SUBSYS          #language en-US "Subsystem info from DHCP"
+#string STR_NVMEOF_SUBSYS_NQN                     #language en-US "  NVM Subsystem NQN"
+#string STR_NVMEOF_SUBSYS_NQN_HELP                #language en-US "The target subsystem's NVMe Qualified Name."
+#string STR_NVMEOF_SUBSYS_ADDRESS                 #language en-US "  NVM Subsystem Address"
+#string STR_NVMEOF_SUBSYS_ADDRESS_HELP            #language en-US "Enter NVM Subsystem address in IPv4,IPv6 or URL format. You need to configure DNS server address in advance if input a URL string."
+#string STR_NVMEOF_SUBSYS_PORT                    #language en-US "  NVM Subsystem Port"
+#string STR_NVMEOF_SUBSYS_NID                     #language en-US "NVM Subsystem NID"

```

Comment:
What's a Subsystem NID?  There's not such thing. It's called a Namespace Identifier.  Just simply call it a "Namespace Identifier (NID)"  Drop the "NVM Subsystem"

Reply:
How about:
```suggestion
#string STR_NVMEOF_SUBSYS_NID                     #language en-US "NVM Namespace Identifier (NID)"
```

Reply:
Tracking this item here:
https://github.com/timberland-sig/edk2/issues/26